app/http.php

```diff
@@ -207,6 +257,8 @@
             $dbForConsole->createCollection('bucket_' . $bucket->getInternalId(), $attributes, $indexes);
         }
 
+        getDomains($dbForConsole, $lastSyncUpdate, $domains, $pools);
+
         $pools->reclaim();
```

Comment:
Let's not reclaim here, as dbForConsole is sent to getDomains, and there we have infinite loop.